### ISSUE TITLE
Dots are not rendered when using FormsAppCompatActivity

### ISSUE THREAD

**Issue by SkyZeroKo** [tomh4/Carousel#1]
Now, I try to use CarouselView but I can't see Dots on page.
Could you tell me how can I resolve this?
![untitled](https://cloud.githubusercontent.com/assets/19888498/17236030/71c0ebc2-5578-11e6-95bb-1020ebac82fa.png)

**Comment by tomh4** [comment]
Yes, I am aware of it. It is because MaterialDesign removes the borders of buttons and therefor the dots cannot be rendered. You can do several things now, one way is to simply use an image as a button and add a gesturehandler, or use a CircleImage ( https://github.com/jamesmontemagno/ImageCirclePlugin  ) wihtout an image, where you can still set a border. This will have the nice effect of making the dots squares :) I will upload a detailed solution and instructions today!

**Comment by SkyZeroKo** [comment]
Thanks for your answer.
I want to use image as a button but I am new to Xamarin,Form development so I don't know how to use.
If you can upload a detailed solution, it ll help me.

**Comment by SkyZeroKo** [comment]
Hi tomh4

I want to create CarouselView using xaml but as I mentioned, I am new member in Xamarin, how can I use CarouselView using Xaml?

Could you kindly teach me about this?

**Comment by tomh4** [comment]
@SkyZeroKo I am not using XAML so i cannot help you there, but check out this link : [Flip through items with Xamarin.Forms CarouselView](https://blog.xamarin.com/flip-through-items-with-xamarin-forms-carouselview/) It explains it really well. 
I updated the project and it now will also work with Material Design, however the Dots turned to Squares. I didnt find a possibility to add Dots that can be colorized so I went with squares.
